src/Enterspeed.Cli/Services/FileService/SchemaFileService.cs

```diff
@@ -216,23 +216,24 @@ private string GetFilePath(string schemaName, string alias, SchemaType schemaTyp
 
         return Path.Combine(schemasDirectoryName, GetFileName(alias, format, schemaType));
     }
-    
+
     private SchemaType GetSchemaTypeFromFilePath(string filePath)
     {
         if (Regex.IsMatch(filePath, ".*.full.(?:js|json)$"))
         {
             return SchemaType.Normal;
         }
+
         if (Regex.IsMatch(filePath, ".*.partial.(?:js|json)$"))
         {
             return SchemaType.Partial;
         }
 
-        if (Regex.IsMatch(filePath, ".*.collection.(?:js)$"))
+        if (Regex.IsMatch(filePath, ".*.partial.(?:js|json)$"))
```

Comment:
Is this a copy paste mistake, I guess it should be collection and not partial?

Reply:
This has now been fixed